assets/blocks/course-outline/style.scss

```diff
@@ -54,68 +41,161 @@ $dark-gray: #1a1d20;
 }
 
 .wp-block-sensei-lms-course-outline-module {
-	border: solid 1px $dark-gray;
-	margin: 1.5em 0;
+	$block: '.wp-block-sensei-lms-course-outline-module';
+
+	&.is-style-default, &.is-style-minimal {
```

Comment:
We only add style-related CSS when a style is set, to make it easier to turn off Sensei CSS in case someone wants to do their own thing.